app/src/main/AndroidManifest.xml

```diff
@@ -58,6 +58,7 @@
             android:launchMode="singleTask"
             android:resizeableActivity="true"
             android:theme="@style/Theme.TermuxActivity.DayNight.NoActionBar"
+            android:excludeFromRecents="true"
```

Comment:
This should only apply after all sessions have exited.
Is that the current behavior with this change?